mrp_brewing/report/report_stock_finished_products.xml

```diff
@@ -72,7 +72,18 @@
                                         <span t-esc="a.origin"/>
                                     </td>
                                     <td>
-                                        <span t-esc="a.raw_material_production_id.brew_order_name"/>
+                                        <t t-if="a.origin">
+                                            <t t-set="lot_number" t-value="False"/>
+                                            <t t-if="a.raw_material_production_id.brew_order_name">
+                                                <t t-set="lot_number" t-value="a.raw_material_production_id.brew_order_name"/>
+                                            </t>
+                                            <t t-if="not a.raw_material_production_id.brew_order_name">
+                                                <t t-foreach="a.quant_ids" t-as="qids">
+                                                    <t t-set="lot_number" t-value="qids.lot_id.display_name"/>

```

Comment:
it means that you're only taking the last one ?

Reply:
from what I remember we told that if there is more than one lot number the lot number of each quant should be put in this td

Reply:
yes, it's the last lot number value that will appear on the report (I noticed it's always the same when multiple lot numbers). If there is a possibility to have different lot numbers, I change the behavior